lib/cartopy/crs.py

```diff
@@ -1815,6 +1815,59 @@ def y_limits(self):
         return self._y_limits
 
 
+class LambertZoneII(Projection):
+    """
+    Lambert zone II (extended) projection (https://epsg.io/27572).
+
+    """
+    def __init__(self):
+        proj4_params = [('proj', 'lcc'),
+                        ('lat_1', 46.8),
+                        ('lat_0', 46.8),
+                        ('lon_0', 0.0),
+                        ('k_0', 0.99987742),
+                        ('x_0', 600000.0),
+                        ('y_0', 2200000.0),
+                        ('pm', 'paris'),
+                        ('units', 'm'),
+                        ('no_defs', None)]
+
+        globe = Globe(semimajor_axis='6378249.2',
+                      semiminor_axis='6356515.0',
+                      towgs84='-168,-60,320,0,0,0,0')
+
+        super().__init__(proj4_params, globe=globe)
+
+        x0, x1, y0, y1 = [0, 1.2e6, 1.6e6, 2.7e6]
```

Comment:
Same question about these bounds values

Reply:
Given the purpose of this projection is to cover hexagonal France and Corsica, these values include those two objects. They do not come from an official standard though. How else could they be chosen? I would gladly welcome suggestions on this and amend accordingly. :)

Reply:
I had a quick look at the EPSG version in your original post, and realized that the boundary there is being set by the `area_of_use`, which should be a good quantity we can reference for our boundary. The interesting thing here is that I think the ellipsoid/globe being used may not correspond to the area of use coordinates being reported.

Here is we are calling `self.as_geodetic()`
https://github.com/SciTools/cartopy/blob/694efcf64f07db5a4b901d1c1fb6d5f32ebb5cd6/lib/cartopy/crs.py#L688
but here it says that it is values reported using the WGS84 ellipse.
https://github.com/SciTools/cartopy/blob/694efcf64f07db5a4b901d1c1fb6d5f32ebb5cd6/lib/cartopy/crs.py#L678

So, I think this may be a bug in the epsg code path and we should be doing something like:
`points = self.transform_points(PlateCarree().as_geodetic(), lons, lats)`

Thoughts on how that looks?

Reply:
Ok, I've done just that. Works just fine. :-)
I have updated the test image too, as the extent is now slightly different from the one I had arbitrarily set before.
Thanks.